CMake/External_FFmpeg.cmake

```diff
@@ -47,6 +52,46 @@ else ()
   set(_FFmpeg_yasm --yasmexe=${fletch_YASM})
   list(APPEND ffmpeg_DEPENDS yasm)
 
+  # Should we try to point ffmpeg at zlib if we are building it?
```

Comment:
The issue is that ffmpeg always seems to link against the system zlib even if we are building our own. I'm not sure if this is a problem or not. 

I'm not sure what happens if no system version is installed. The reason that I mentioned this is that when I build fletch with ffmpeg and zlib, and then I `ldd` the built libraries, I see that the ffmpeg libs point to the system version `libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1`, instead of `libz.so`, which is in the same build directory. 

I think this behavior is fine if we aren't building zlib, but should FFmpeg be pointing to the system version of the library if we are building it?